### ISSUE TITLE
Drush 9: Site Archive Removed?

### ISSUE THREAD

**Issue by nickschuch** [drush-ops/drush#3003]
Hey hey!

Have been doing some testing with Drupal 8.4 and noticed that `site-archive` was removed in Drush 9.

https://github.com/drush-ops/drush/commit/8beffa74f0cbf0359f318f9080bbbe59d6a25d9b

I was wondering if you were looking for contributions to add it back?

**Comment by weitzman** [comment]
Yes please. See https://github.com/drush-ops/drush/issues/1779, and dont
miss the tip in last comment. Thanks.
